tests/e2e/framework/framework.go

```diff
@@ -0,0 +1,192 @@
+package framework
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	"io/ioutil"
+	"os"
+	"sync"
+)
+
+var (
+	logsBucketPath = flag.String("logs_bucket_path", "", "Cloud Storage Bucket path to use to store logs")
+)
+
+const (
+	TMP_PREFIX = "istio.e2e."
+)
+
+type TestInfo struct {
```

Comment:
Comments on all public symbols.

This brings up the issue that we should be running the linters on code in this repo. Otherwise, bugs will slip in.